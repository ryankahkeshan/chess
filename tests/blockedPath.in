game human human
move d1 f3
move e2 e4
move h8 h4
move f8 h6
move a8 a5
move c8 e6
move d8 d7
move g8 f6
resign
