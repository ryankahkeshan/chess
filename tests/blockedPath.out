8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _ _ _
3 _ _ _ _ 
2 PPPPPPPP
1 RNBQKBNR

  abcdefgh

Invalid move
8 rnbqkbnr
7 pppppppp
6  _ _ _ _
5 _ _ _ _ 
4  _ _P_ _
3 _ _ _ _ 
2 PPPP PPP
1 RNBQKBNR

  abcdefgh

Invalid move
Invalid move
Invalid move
Invalid move
Invalid move
8 rnbqkb r
7 pppppppp
6  _ _ n _
5 _ _ _ _ 
4  _ _P_ _
3 _ _ _ _ 
2 PPPP PPP
1 RNBQKBNR

  abcdefgh

Black wins!
Final Score:
White: 0
Black: 1
